WalletWasabi/Blockchain/Analysis/FeesEstimation/HybridFeeProvider.cs

```diff
@@ -158,4 +166,14 @@ private bool SetAllFeeEstimate(AllFeeEstimate fees)
 		AllFeeEstimate = fees;
 		return true;
 	}
+
+	public async Task<int?> FetchTransactionFeeAsync(uint256 txId)
```

Comment:
In my mind, this is the method that the ViewModel calls. This is a heavy operation! It can easily take up to ~15 seconds to fetch the fee. It's slow bc it's called over Tor.